src/artwork.c

```diff
@@ -392,6 +393,64 @@ static struct online_source musicbrainz_source =
   };
 
 
+/* -------------------------------- HELPERS -------------------------------- */
+
+#define TMP_BUF_SIZE 1024
+static int artwork_read_byurl(struct evbuffer *evbuf, const char *url);
+
+/* Reads an artwork url tags from the json responce within evbuffer, then if found 
+ * makes call to read image from url and replace evbuffer contents with image data
+ *
+ * @out evbuf     Json data passed | replaced with Image data returned
+ * @in  url       URL for the image
+ * @return        ART_FMT_* on success, ART_E_ERROR otherwise
+ */
+
+static int
+artwork_read_json_byurl(struct evbuffer *evbuf, const char *url)
+{
+  int ret = ART_E_ERROR;
+  json_object *jresponse;
+  char *body;
+  cfg_t *lib;
+  int i, n;
+
+  evbuffer_add(evbuf, "", 1);
+  body = (char *)evbuffer_pullup(evbuf, -1);
+  
+  DPRINTF(E_SPAM, L_ART, "Response from '%s': %s\n", url, body);
+    
+  jresponse = json_tokener_parse(body);
+  if (!jresponse) {
+    DPRINTF(E_LOG, L_ART, "Artwork from '%s' has unknown json\n", url);
+    jparse_free(jresponse);
+  } else {
+    lib = cfg_getsec(cfg, "library");
+    n = cfg_size(lib, "stream_urlimage_tags");
+    for (i = 0; i < n; i++) {
+        //DPRINTF(E_SPAM, L_ART, "Artwork from '%s' searching for tag '%s'\n", url, cfg_getnstr(lib, "stream_urlimage_tags", i));
+      const char *aurl = jparse_str_from_obj(jresponse, cfg_getnstr(lib, "stream_urlimage_tags", i));
+      if (aurl) {
+        char buf[TMP_BUF_SIZE];
```

Comment:
Please study the coding standard of the rest of forked-daapd and use the same - i.e. the wacky indentation, braces, declarations